source/Generic/GamesSizeCalculator/GamesSizeCalculatorSettings.cs

```diff
@@ -25,6 +26,42 @@ public class GamesSizeCalculatorSettings : ObservableObject
         public bool IncludeDlcInSteamCalculation { get => includeDlcInSteamCalculation; set => SetValue(ref includeDlcInSteamCalculation, value); }
         private bool includeOptionalInSteamCalculation = false;
         public bool IncludeOptionalInSteamCalculation { get => includeOptionalInSteamCalculation; set => SetValue(ref includeOptionalInSteamCalculation, value); }
+
+        private bool getUninstalledGameSizeFromGog = true;
+        public bool GetUninstalledGameSizeFromGog { get => getUninstalledGameSizeFromGog; set => SetValue(ref getUninstalledGameSizeFromGog, value); }
+
+        private bool getSizeFromGogNonGogGames = true;
+        public bool GetSizeFromGogNonGogGames { get => getSizeFromGogNonGogGames; set => SetValue(ref getSizeFromGogNonGogGames, value); }
+
+        private ObservableCollection<string> depotRegionWords = new ObservableCollection<string> { "eu", "europe", "row", "en", "english", "ww" };
+        public ObservableCollection<string> DepotRegionWords { get => depotRegionWords; set => SetValue(ref depotRegionWords, value); }
+
+        private ObservableCollection<string> depotRegionWordsBlacklist = new ObservableCollection<string>
+        {
+            "asia",
+            "aus",
+            "australia",
+            "nz",
```

Comment:
Is this collection modifiable by the user or will it never change? If it never changes then it should be moved to the settings Viewmodel object instead as there wouldn't be a purpose in these being here. If they are modifiable then ignore this